cantools/database/__init__.py

```diff
@@ -366,24 +372,32 @@ def load_can_database(fmt: str) -> can.Database:
             return load_can_database('arxml')
         except (ElementTree.ParseError, ValueError) as e:
             e_arxml = e
+            if debug:
```

Comment:
Maybe we could just raise UnsupportedDatabaseFormatError here with the debug flag?

Reply:
good suggestion (didn't know this trick yet). Thanks for the hint!

I've implemented it this way and IMO the result is quite a bit cleaner and shorter now...